docs/index.html

```diff
@@ -357,6 +367,68 @@ <h5>Browser extensions</h5>
         </dd>
     </dl>
 
+    <h5>SSH Agent</h5>
+    <dl>
+        <dt id="faq-ssh-agent-how"><a href="#faq-ssh-agent-how">How does the SSH Agent work?</a></dt>
+        <dd>
+            The SSH Agent feature is supported on all target platforms (Linux, macOS and Windows) and it acts as a client
+            for an existing agent despite its name. It can automatically add SSH keys from your KeePassXC database to a
+            running SSH agent when unlocked and remove when locked.<br>
+            <br>
+            On Linux, most desktops are already running an agent without any set up required.<br>
+            On Windows, you need to have <i>Pageant</i> running. It is part of the
+            <a href="https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html">PuTTY suite</a>.<br>
+            On macOS, <code>ssh-agent</code> is running by default and no further setup is required.
+        </dd>
+    </dl>
+    <dl>
+        <dt id="faq-ssh-agent-keys"><a href="#faq-ssh-agent-keys">What SSH key types are supported?</a></dt>
+        <dd>
+            Most major key types are supported (DSA, RSA and Ed25519), including encrypted keys. ECDSA keys are only
+            supported in new OpenSSH file format. Really old 3DES encrypted keys are not and such keys are encouraged
+            to be upgraded for external storage or decrypted for in-database use.<br>
```

Comment:
Just "Encrypted 3DES keys are not supported and we highly recommend upgrading them"

Reply:
You can't say "Encrypted 3DES keys" as 3DES is the cipher and not the key.

Reply:
Err... Yes. Of course. I meant "3DES-encrypted keys are not supported and we highly recommend upgrading them".
/me needs his morning coffee.